DatadogRUM/Sources/RUMMonitor/Scopes/RUMApplicationScope.swift

```diff
@@ -71,7 +77,7 @@ internal class RUMApplicationScope: RUMScope, RUMContextProvider {
 
         // Create the application launch view on any command
```

Comment:
The hope was that these new requirements would remove some of the complexity here -- not waiting on a command and not requiring the RUMSDKInitCommand anymore.  Is that not the case?

Reply:
Unless we implement "non interactive sessions", removal of `RUMSDKInitCommand` seems impossible to me without adding new (temporary) complexity. I wish we could remove this command today, but existing logic still requires starting initial view lazily in certain situations (background + BET enabled). 

But I totally agree: ideally, we should remove `RUMSDKInitCommand` and create `RUMSessionScope` with initial `RUMViewScope` from `RUM.Monitor.init()` (🏅). This is mid-term goal to me, not feasible today IMO.

Does it make sense @fuzzybinary ?